engine/engine.go

```diff
@@ -294,7 +294,11 @@ func initialize(dbp zesty.DBProvider, publicID string) (*resolution.Resolution,
 		}
 		// task validation error
 		// -> possible race condition with updated template
-		// put task on hold
+		// put task + resolution on hold
+		res.SetState(resolution.StateCrashed)
```

Comment:
CRASHED seems weird to me.
Looking at the current collectors I don't see any of them auto picking CRASHED resolutions, but it could certainly happen at some point.
This on the other hand doesnt seem very retryable, there's a mismatch.
Since we're putting the task on BLOCKED, I'd lean towards a flavor of BLOCKED_XXX for the resolution itself. BLOCKED_FATAL or BLOCKED_BADREQUEST come to mind.

Reply:
What about ?
```suggestion
		res.SetState(resolution.StateBlockedToCheck)
```